tensorboard/plugins/text/http_api.md

```diff
@@ -0,0 +1,37 @@
+# Text plugin HTTP API
+
+The text plugin name is `text`, so all its routes are under
+`/data/plugin/text`.
+
+## `/data/plugin/text/tags`
+
+Retrieves an index of tags containint text data.
+
+Returns a dictionary mapping from `runName` (quoted string) to an
+array of `tagName`. Here is an example:
+
+  {
+    "text_demo_run": [
+      "simple_example/greeting",
+      "markdown_table/chocolate_study",
+      "higher_order_tensors/multiplication_table"
+    ]
+  }
+
+Runs without any text tags are omitted from the result.
+
+## `/data/plugin/text/text?run=foo&tag=bar`
+
+Returns an array of text events for the given run and tag.  Each event is
+a dictionary with members `step`, `text`, and `wall_time`, where `wall_time` is
+a floating-point number of seconds since epoch, `step` is an integer step
+counter, and `text` is the actual HTML for rendering the text data
+on the frontend.
+
+Example:
+
+  {
+    "step": 4
+    "text": "<table>↵<tbody>↵<tr>↵<td><p>....</p></td>↵</tr>↵</tbody>↵</table>"
+    "wall_time": 1591289315.827554
+  }
```

Comment:
put this in a code block
``````
```json
 {
    "step": 4,
    "text": "<table>↵<tbody>↵<tr>↵<td><p>....</p></td>↵</tr>↵</tbody>↵</table>",
    "wall_time": 1591289315.827554
  }
```
``````

It should look reasonable in GitHub when you click on this: 
![image](https://user-images.githubusercontent.com/2547313/85156269-af62de80-b20e-11ea-8cdf-0cd49b444d50.png)

Reply:
got it, done